packages/flutter/test/rendering/independent_layout_test.dart

```diff
@@ -32,8 +34,8 @@ class TestLayout {
 void main() {
   TestRenderingFlutterBinding.ensureInitialized();
 
-  const ViewConfiguration testConfiguration = ViewConfiguration(
-    size: Size(800.0, 600.0),
+  final ViewConfiguration testConfiguration = ViewConfiguration(
+    constraints: ViewConstraints.tight(const Size(800.0, 600.0)),
```

Comment:
Seems like `ViewConstraints.tight` could be `const`, allowing this to be const.

Reply:
Sadly, Dart currently doesn't allow `ViewConstraints.tight` to be const, see https://github.com/flutter/engine/pull/48090/files#r1406511737.